### PR TITLE
Fix Chinese content & missing string

### PR DESCRIPTION
**Pull request by Dissyu**
Fix some Chinese punctuation & content, add new string with compact_adr97.

### CODE REVIEW

File: optionals/compat_adr_97/stringtable.xml [hunk 1]
```diff
@@ -15,6 +15,8 @@
             <Japanese>P90 TR (黒)</Japanese>
             <TChinese>P90 TR (黑色)</TChinese>
```
Comment (bux) [inline optionals/compat_adr_97/stringtable.xml]: @jonpas didn't you replace all stringtable entries?

Comment (jonpas) [inline optionals/compat_adr_97/stringtable.xml]: Oh, maybe the compats were missed. :/

**Comment by Dissyu** [comment]
Remove adr97 unnecessary string, this time might cleaner

File: optionals/compat_adr_97/stringtable.xml (hunk 1 above)
Comment (Dissyu) [inline optionals/compat_adr_97/stringtable.xml]: already fix~~~ @jonpas Thanks you remind me~~

**Comment by jonpas** [comment]
Are you and @GodofMonkeys working together or would you prefer you look at each other's changes before we merge them?

**Comment by Dissyu** [comment]
@jonpas ok, we merge together.

**Comment by jonpas** [comment]
Not what I meant, was just asking. We can merge them separately. I was just asking in case you don't agree on some translations.

**Comment by GodofMonkeys** [comment]
@jonpas That's ok, we compare it, don't have any conflict with any string. It is good to go :)

**Comment by jonpas** [comment]
Awesome! Keep them coming. ;)

**Pull request merged**